File: src/C#/CreationalPatterns/AbstractFactory/EraStyles/Products/RenaissanceClothing.cs
using EraStyles.Abstraction;
using EraStyles.Constants;
using System;
using System.Collections.Generic;
using System.Text;

namespace EraStyles.Products
{
    public class RenaissanceClothing : EraObject
    {
        public override void ShowDetails()
        {
            Console.WriteLine(string.Format(GlobalConstants.ClothingMessage, "Renaissance"));
        }
    }
}

File: src/JavaScript/CreationalPatterns/AbstractFactory/EraStyles/products/renaissanceClothing.js

const EraObject = require("../abstraction/eraObject");
const GlobalConstants = require("../constants/globalConstants");

class RenaissanceClothing extends EraObject {
    showDetails() {
        console.log(GlobalConstants.ClothingMessage.replace("{0}", "Renaissance"));
    }
}

module.exports = RenaissanceClothing;